docs/architecture/modern-web-apps-azure/choose-between-traditional-web-and-single-page-apps.md

```diff
@@ -35,7 +35,9 @@ Improvements in user experience made possible by the SPA approach must be weighe
 
 ## Blazor
 
-ASP.NET Core 3.0 introduces a new model for building rich, interactive, and composable UI called Blazor. Blazor server-side allows developers to build UI with Razor on the server and for this code to be delivered to the browser and executed client-side using [WebAssembly](https://webassembly.org/). Blazor server-side is available now with ASP.NET Core 3.0 or later. Blazor client-side should be available in 2020.
+ASP.NET Core 3.0 introduces a new model for building rich, interactive, and composable UI called Blazor. Blazor server-side allows developers to build UI with C# and Razor on the server and for the UI to be interactively connected to the browser in real-time using a persistent SignalR connection.
+
+Blazor WebAssembly introduces another option for Blazor apps, allowing them to run in the browser using WebAssembly. Because it's real .NET running on WebAssembly, you can re-use code and libraries from server-side parts of your application.
```

Comment:
Note in passing ... A Blazor WebAssembly app (standalone) doesn't have "server-side parts," so this description is a little off. The phrase "server-side parts of your application" implies that such apps always have "server-side" parts. However, I understand that readers will need to get into the hosting model details in the Blazor docs to understand it further. There's no important reason to spend a lot of time here with the concepts.